yarn.lock

```diff
@@ -1165,13 +1165,20 @@
   dependencies:
     regenerator-runtime "^0.13.4"
```

Comment:
is this file needed in the repo?

Reply:
the create react app process created the initial scaffolding, including a .gitignore file. I suspect it is not required, and perhaps something we should remove in future commits.

Reply:
alright, we can investigate later